docs/step-by-step/step-07.md

```diff
@@ -0,0 +1,108 @@
+
+## Check if there's an update to Netdata
+
+You can always check if there is an update available from the **Update** area of your Netdata dashboard.
+
+![Animated GIF of opening the update
+modal](https://user-images.githubusercontent.com/12263278/64876743-be957a00-d647-11e9-83dd-2f0a8df572cb.gif)
+
+If an update is available, you'll see a modal similar to the one above.
+
+When you use the [automatic one-line installer script](../../packaging/installer/README.md#one-line-installation),
+Netdata will automatically attempt to update every day. If you choose to update it manually, there are [several
+well-documented methods](../../packaging/installer/UPDATE.md) to achieve that. However, it is best practice for you to
+first go over the [changelog](../../CHANGELOG.md).
+
+## Export and import a snapshot
+
+Netdata can export and import snapshots of the contents of your dashboard at a given time. Any Netdata agent can import
+a snapshot created by any other Netdata agent.
+
+Snapshot files include all the information of the dashboard, including the URL of the origin server, its unique ID, and
+chart data queries for the visible timeframe. While snapshots are not in real-time, and thus won't update with new metrics, you can still pan, zoom, and highlight charts as you see fit.
+
+Snapshots can be incredibly useful for diagnosing anomalies after they've already happened. Let's say Netdata triggered
+an alarm while you were sleeping. In the morning, you can look up the exact moment the alarm was raised, export a
+snapshot, and send it to a colleague for further analysis.
+
+To export a snapshot, click on the **export** icon.
+
+![Animated GIF of opening the export
+modal](https://user-images.githubusercontent.com/12263278/64901454-48b60080-d691-11e9-9c14-1539bc841735.gif)
+
+Edit the snapshot file name and select your desired compression method. Click on **Export**.
+
+When the export is complete, your browser will prompt you to save the `.snapshot` file to your machine. You can now
+share this file with any other Netdata user via email, Slack, or even to help describe your Netdata experience when
+[filing an issue](https://github.com/netdata/netdata/issues/new/choose) on GitHub.
+
+To import a snapshot, click on the **import** icon.
+
+![Animated GIF of opening the import
+modal](https://user-images.githubusercontent.com/12263278/64901503-ee696f80-d691-11e9-9678-8d0e2a162402.gif)
+
+Select the Netdata snapshot file to import. Once the file is loaded, the dashboard will update with critical information
+about the snapshot and the system from which it was taken. Click **import** to render it.
+
+Your Netdata dashboard will load data contained in the snapshot into charts. Because the snapshot only covers a certain
+period, it won't update with new metrics.
+
+An imported snapshot is also temporary. If you reload your browser tab, Netdata will remove the snapshot data and
+restore your real-time dashboard for your machine.
```

Comment:
I think we should probably explicitly call out here that loading a snapshot from an untrusted source is risky. It's not immediately obvious to that this might be the case to people who don't understand in detail how snapshots wor

Reply:
Could you elaborate a bit? Or are you talking about general security best practices, like not plugging in USB drives you find on the sidewalk?

Reply:
Last I knew, snapshots included some JS code that gets executed on load, and we kind of just trust that the code in question is not malicious.

Assuming I'm correct about my understanding of snapshots, that could get all the way to arbitrary code execution if the user has a sufficiently insecure browser.

Even aside from that though, we're parsing a very large amount of untrusted data when we load a snapshot, and I'm not sure that the relevant code has been properly audited.

Reply:
Got it. Added a blockquote to address this without getting too specific.